packages/sui-test/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@s-ui/test",
+  "version": "1.0.0-beta.10",
+  "description": "",
+  "bin": {
+    "sui-test": "./bin/sui-test.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@s-ui/helpers": "1.5.0",
+    "babel-core": "6.26.0",
+    "babel-polyfill": "6.26.0",
+    "babel-preset-sui": "1.5.0",
```

Comment:
Same. In this case I think is particularly important to be consistent with the code we're testing.

Reply:
👍